Source/WebKit/Platform/IPC/Connection.h

```diff
@@ -279,11 +279,33 @@ class Connection : public ThreadSafeRefCounted<Connection, WTF::DestructionThrea
             return WTFMove(replyArguments).value();
         }
 
-        template<typename... U>
+        template<typename U> struct ValidInvocableReplyArgumentsConstructorType
```

Comment:
I'm a bit hesitant to add half a screen of template soup just to replace the call site `if` statements for what amounts to mostly returning `emptyString()`.. 
How would you feel about just direct call site if statements?